idaes/core/util/misc.py

```diff
@@ -32,23 +32,26 @@
 
 
 # Author: Andrew Lee
-def get_solver(solver=None, options={}):
+def get_solver(solver=None, options=None):
     """
     General method for getting a solver object which defaults to the standard
     IDAES solver (defined in the IDAES configuration).
 
     Args:
         solver: string name for desired solver. Default=None, use default solver
         options: dict of solver options to use, overwrites any settings
-                 provided by IDAES configuration.
+                 provided by IDAES configuration. DEfault = None.
```

Comment:
the extra "default=None" here is either redundant or needs to be explained.

Reply:
@ksbeattie I added an explanation here, and in all the other places where a similar statement was made. However, this means I need a review again.